app/templates/partials/user_profile/middle_content.html

```diff
@@ -3,6 +3,7 @@
   </div>
   <!-- Upcoming events -->
   <div class="mb-3">
+    <h3>Upcoming events</h3>
```

Comment:
Are there h1 and h2 before? If not - use the highest unused title.
To reach a certain visual size, use CSS.

Reply:
done